src/Deal/DealRepository.php

```diff
@@ -50,9 +51,11 @@ private function createNewDeal(
 
     /**
      * @param array<mixed> $jsonFromBody
+     * @param array<Event> $eventsWithAccess
      */
     public function trySaveNewDealFromGoogleForm(
         array $jsonFromBody,
+        array $eventsWithAccess
```

Comment:
přidat trailing comma
```suggestion
        array $eventsWithAccess,
```